.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+**Reference to a related issue in the repository**
+Add a reference to a related issue in the repository.
+
+**Add a description**
+Add a description of the changes proposed in the pull request.
+
+Some questions to ask
+What is this change?
+What does it fix?
+Is this a bugfix or a feature? Does it break any existing functionality or force me to update to a new version?
+How has it been tested?
+
+**Take this checklist as orientation for yourself, if this PR is ready for Maintainer Review**
+- [ ] My suggestion follows the [governance rules](https://github.com/openMSL/governance-and-documentation).
+- [ ] I have taken care about the [message documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_messages.html) and the [fields and enums documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_fields_enums.html).
```

Comment:
Is this important for OpenMSL? Actually the ASAM OSI should be cloned and included and therefore the rules are already applied. What was your idea to add this here?

Reply:
Sorry, Copy - Paste error